consul/src/main/scala/io/buoyant/consul/v1/KvApi.scala

```diff
@@ -61,6 +63,30 @@ class KvApi(
   }
 
   /**
+   * Get key(s)
+   *
+   * https://www.consul.io/docs/agent/http/kv.html#single
+   *
+   * @param path path to the key, must start with /
+   */
+  def multiGet(
+    path: String,
+    datacenter: Option[String] = None,
+    blockingIndex: Option[String] = None,
+    recurse: Boolean = false,
```

Comment:
> for multiGet, recurse should always be true, right?

I'd say it _can_ but not _must_. Do you mind leaving it optional?
I expect usage as:
```
def getNs(path: String, index: Option[String] = None) =
    api.get(path, blockingIndex = index, recurse = path.endsWith("/"))
```

so it'd be much more convenient to switch the `recurse` flag rather than use another method.